tools/run_parallel.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Uncommenting the next line may be useful when debugging this script.
+#set -x
+
+usage () {
+    cat << EOF
+Usage: run_parallel.sh {command} {num_iter} {num_parallel}
```

Comment:
Suggest adding "[ ]" around the last parameter `{num_parallel}` to signal it's optional.

Reply:
I've done this too.